config/esp32/args.gni

```diff
@@ -29,6 +29,9 @@ lwip_platform = "external"
 chip_inet_config_enable_tcp_endpoint = false
 chip_inet_config_enable_udp_endpoint = true
 
+#Use platform memory management
```

Comment:
This comment just re-states the code below.
Could you write something that aids in code reading rather than re-stating it? Maybe why this is the correct choice? otherwise just remove the comment.

Reply:
Removed the comment